docs/example/searchbarDocs.vue

```diff
@@ -0,0 +1,150 @@
+<template>
+  <doc-section
+    id="searchbarDocs"
+    name="searchbar">
```

Comment:
- As @yamgent [suggested](https://github.com/MarkBind/vue-strap/pull/30#discussion_r190791591), let's modify **.eslintrc.js** (it should be committed):

  ```js
  'vue/max-attributes-per-line': [
    'error',
    {
      'singleline': 2,
      'multiline': {
        'max': 1,
        'allowFirstLine': false,
      },
    },
  ],
  ```

  Let's keep it strict.

- Capitalize `S` in `name="searchbar"`:

  ```html
  <doc-section id="searchbarDocs" name="Searchbar">
  ```